dashboard/test/models/pd/application/facilitator1819_application_test.rb

```diff
@@ -257,5 +259,34 @@ class Facilitator1819ApplicationTest < ActiveSupport::TestCase
       @application_with_fit_workshop.update! fit_workshop_id: nil
       refute @application_with_fit_workshop.registered_fit_workshop?
     end
+
+    test 'workshop cache' do
+      create :pd_enrollment, workshop: @fit_workshop, user: @application_with_fit_workshop.user
+
+      # Original query: Workshop, Session, Enrollment
+      assert_queries 3 do
+        assert_equal @fit_workshop, @application_with_fit_workshop.fit_workshop
+      end
+
+      # Cached
+      assert_queries 0 do
```

Comment:
is it worth doing time manipulation to see that the cache is not used after 30 seconds?